fzf-tab.zsh

```diff
@@ -123,15 +128,30 @@ _fzf_tab_get_candidates() {
                 dsuf=/
             fi
         fi
-        candidates+=$k$'\0'$dsuf
+
+        # add color to description if they have group description
+        if [[ $v[expl] ]]; then
+            local index=$_fzf_tab_groups[(ie)$v[expl]]
+            local color=$FZF_TAB_GROUP_COLOR[$index]
+            # add a hidden group index at start of string to keep group order when sorting
+            # FIXME: only support 16 groups
+            candidates+=$(( [##16] index ))$'\b'$color$'\0'$k$'\0'$dsuf$reset_color
+        else
+            candidates+=1$'\b\0'$k$'\0'$dsuf
+        fi
     done
     (( same_word )) && candidates[2,-1]=()
     local LC_ALL=C
     candidates=("${(@on)candidates}")
+
+    for k in {1..$#_fzf_tab_groups}; do
+        headers+=$FZF_TAB_GROUP_COLOR[k]$_fzf_tab_groups[k]$reset_color
```

Comment:
You need to autoload and call `colors` for `reset_color` to become available. However, it's better to hard-code $\033[00m` since you've hard-coded the other escape codes anyway.

Reply:
Oh, I removed `colors` in the second commit but forgot to expand it. Thanks.